.gitignore

```diff
@@ -255,4 +255,4 @@ doc-nits
 
 # LSP (Language Server Protocol) support
 .cache/
-compile_commands.json
+compile_commands.json
```

Comment:
Please add back the end of line character.

Reply:
Done